jmslib/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>checker-qual</artifactId>
         </dependency>
 
+	<dependency>
+	    <groupId>org.apache.qpid</groupId>
+	    <artifactId>qpid-jms-client</artifactId>
+            <version>0.56.0</version>
+	</dependency>
```

Comment:
There is a property for qpid jms client version at https://github.com/rh-messaging/cli-java/blob/master/cli-qpid-jms/pom.xml#L38. We certainly dont want to hardcode the version like this. It would not work for .redhat-00001 builds then.

Reply:
could you possibly guide me how to access that property from this pom file?

Reply:
You can't, we'd have to move the property where it is visible. And you shouldn't. We don't want to compile qpid-jms into every cli we create. You should either match exception text or do the other thing I described in my review comment.

Reply:
but without that, you won't be able to import JmsConnectionRemotelyClosedException org.apache.qpid.jms. Thus you won't be able to catch that exception, so that makes this idea unusable. thoughts?

Reply:
ok, I still can use the ugly exception text match. I didn't noticed your comment at the bottom.

Reply:
Or you can do the two other things, but they are harder. Checking class name with reflection might not be that hard...